capa/rules/cache.py

```diff
@@ -26,6 +28,58 @@
 CacheIdentifier = str
 
 
+def is_dev_environment() -> bool:
+    if getattr(sys, "frozen", False):
+        # running as a PyInstaller executable
+        return False
+
+    if "site-packages" in __file__:
+        # running from a site-packages installation
+        return False
+
+    if not shutil.which("git"):
+        # git is found, but might not be always be in PATH
+        # we should handle this case
+        return False
```

Comment:
should we check for existence of a `.git` directory instead?

Reply:
to better tell this is a copy installed from the repo

Reply:
and then check if git is installed afterwards (btw. does this work across OSs?)

Reply:
`shutil` is part of Python standard library. I tested it on Windows, LInux and MacOS. I think it is good.

Reply:
noted, thanks